### PR TITLE
[#736] Fixes Nightwatch setup and instructions, and re-enables in Cir…

### PR DESCRIPTION
**Pull request by justafish**
…cleCI

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/736

## Testing instructions

- Check that nightwatch can be run locally by following the instructions (check your `.env.local` isn't overriding things erroneously)
- Check that the tests pass on CircleCI

### CODE REVIEW

File: packages/admin-ui/package.json [hunk 1]
```diff
@@ -91,8 +91,7 @@
     "nightwatch": "nightwatch --config ./src/tests/Nightwatch/nightwatch.conf.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "// Test commands for the CI": "⬅✌➡",
-    "test": "yarn test:unit",
-    "test:functional": "yarn nightwatch",
```
Comment (dawehner) [inline packages/admin-ui/package.json]: It feels like it would be nice to be able to run unit tests independent from really slower functional tests

Comment (justafish) [reply packages/admin-ui/package.json]: @dawehner the unit tests are so fast I don't think it's worth splitting into separate jobs currently

Comment (dawehner) [reply packages/admin-ui/package.json]: That makes sense, thank you

Comment (martinfrances107) [reply packages/admin-ui/package.json]: I think this PR is good improvement.

I have tried the caution approach of nuking my setup and starting from scratch, and then pig headily trying all the commands.
 
a) I about to file a separate issue but

 yarn workspace @drupal/admin-ui build

failed  -  reporting that a script could not run 'react-scripts'  the solution to this was to run yarn install - and ensure it was available. We have just had a bump in react-scripts version number committed. So something for me to look into elsewhere.

b) With a "I'm trying to break things hat on".. can I ask for a minor touch up to the text of readme.md

there seem to be two candidate locations for .env.local
in the root directory "admin-ui"

or 

in the package directory "admin-ui/packages/admin-ui"

The text does not say which.

I think if we "time and motioned" 100 developers we would see confusion on this point..., Maybe I am projecting.

c) Lastly I never have or can now get the nightwatch tests to pass.  I see error messages to the effect of looking for visually hidden element but element not found  - timed out.

So what I am really saying is this PR is almost ready ... If I ever get to the bottom on my problems and it is helpful to the wider community I will create a new issue - but I did not want to given the PR a clean bill of health with that caveat.

File: README.md [hunk 2]
```diff
@@ -57,18 +55,21 @@ Open the one time login link in your browser to log into Drupal. You will then h
...
+
 #### Nightwatch
 - If you don't know the password for admin, change it with `docker exec -it drupal_admin_ui_drupal drush user:password admin admin`
-- Run `docker exec -it -e REACT_APP_DRUPAL_BASE_URL=http://drupal drupal_admin_ui_node yarn workspace admin-ui build`, which
-creates a build that uses `http://drupal` as it's base URL, which is the URL that the node container sees Drupal on internally.
-- Run `docker exec -it -e REACT_APP_DRUPAL_BASE_URL=http://drupal -e NIGHTWATCH_LOGIN_admin_PASSWORD=admin drupal_admin_ui_node yarn workspace admin-ui nightwatch`
-or `docker exec -it -e REACT_APP_DRUPAL_BASE_URL=http://drupal -e NIGHTWATCH_LOGIN_admin_PASSWORD=admin drupal_admin_ui_node yarn test` to run all tests
-- When you're finished, restore the regular build if you want to browse the compiled version in your browser with `docker exec -it drupal_admin_ui_node yarn workspace admin-ui build`.
-This will also be restored when you restart your containers.
+- Update your `.env.local` file, setting `NIGHTWATCH_LOGIN_admin_PASSWORD` to the password you set above e.g. `NIGHTWATCH_LOGIN_admin_PASSWORD=admin`
```
Comment (dawehner) [inline README.md]:
```suggestion
- Update your `.env.local` file, setting `NIGHTWATCH_LOGIN_admin_PASSWORD` to the password you set above e.g. `NIGHTWATCH_LOGIN_admin_PASSWORD=admin`
- If you want to test against the current JS, not the production build change set `NIGHTWATCH_URL=http://127.0.0.1:3000` in the `.env.local` file.
```
If you

**Pull request merged**